src/main/java/com/twilio/rest/api/v2010/account/CallReader.java

```diff
@@ -127,46 +131,21 @@ public CallReader setStatus(final Call.Status status) {
      * `StartTime&gt;=YYYY-MM-DD` to read calls that started on or after midnight of
      * this date..
      *
-     * @param absoluteStartTime Only include calls that started on this date
+     * @param startTime Only include calls that started on this date
      * @return this
      */
-    public CallReader setStartTime(final DateTime absoluteStartTime) {
-        this.rangeStartTime = null;
-        this.absoluteStartTime = absoluteStartTime;
+    public CallReader setStartTime(final ZonedDateTime startTime) {
+        this.startTime = startTime;
         return this;
     }
 
-    /**
-     * Only include calls that started on this date. Specify a date as `YYYY-MM-DD`
-     * in GMT, for example: `2009-07-06`, to read only calls that started on this
-     * date. You can also specify an inequality, such as `StartTime&lt;=YYYY-MM-DD`,
-     * to read calls that started on or before midnight of this date, and
-     * `StartTime&gt;=YYYY-MM-DD` to read calls that started on or after midnight of
-     * this date..
```

Comment:
What happened to these docs? They provide context for the difference between startTime and startTimeBefore, etc.

Reply:
They're up above on `setStartTime`. It doesn't _really_ make sense on any of these methods specifically since it's talking about `StartTime&gt;=YYYY-MM-DD` and the like.

Reply:
I think some clarification that the `Before()` maps to `StartTime&lt;=YYYY-MM-DD` and `After` maps to `StartTime&gt;=YYYY-MM-DD` would be helpful in these situations. Yes the docs say that you _can_ specify the inequality, but they're not on the methods that actually allow you to use that functionality.

Reply:
The doc string is coming from the api definition.

Reply:
Yeah, but we control the building of these additional functions. We can make the inequality setter builders render a clarifying function docstring ourselves that is independent of api-defs

Reply:
I will be looking at these comments again after implementing the Yoyodyne changes to ensure they make sense in context.